examples/basic/main.go

```diff
@@ -0,0 +1,178 @@
+		panic("Expire response should be true/successful")
+	} else {
+		fmt.Println("-----------------------EXPIRE-----------------------")
+		fmt.Println("Successfully set expiration for key \"" + key + "\" with response \"" + strconv.FormatBool(expResp.Val()) + "\"")
+		fmt.Println("-----------------------EXPIRE-----------------------")
+	}
+	fmt.Println()
+	ttlResp := client.TTL(ctx, key)
+	if ttlResp.Val() == -2 {
+		panic("Received no key exists response for existing key \"" + key + " while fetching TTL")
+	} else {
+		fmt.Println("-----------------------TTL-----------------------")
+		fmt.Println("Successfully received remaining ttl " + ttlResp.Val().String() + " for key \"" + key + "\"")
+		fmt.Println("-----------------------TTL-----------------------\n")
+	}
+
+	if !useRedis {
+		momentoCacheClient.DeleteCache(ctx, &momento.DeleteCacheRequest{
+			CacheName: momentoCacheName,
+		})
+	}
+
+}
+
+func initMomentoRedisClient(options MomentoOptions) *momentoredis.MomentoRedisClient {
+	momentoCacheName = options.cacheName
+	credential, eErr := auth.NewEnvMomentoTokenProvider("MOMENTO_AUTH_TOKEN")
+	if eErr != nil {
+		panic("Failed to initialize credentials through auth token. Did you export the environment" +
+			" variable MOMENTO_AUTH_TOKEN?\n" + eErr.Error())
+	}
+	cacheClient, cErr := momento.NewCacheClient(config.LaptopLatest(), credential, time.Duration(options.defaultTTlSeconds)*time.Second)
+	if cErr != nil {
+		panic("Failed to initialize Momento cache client\n" + cErr.Error())
+	}
+	// create cache; it resumes execution normally incase the cache already exists and isn't exceptional
+	cacheClient.CreateCache(context.Background(), &momento.CreateCacheRequest{
+		CacheName: options.cacheName,
+	})
+
+	redisClient := momentoredis.NewMomentoRedisClient(cacheClient, options.cacheName)
+	momentoCacheClient = cacheClient
+	return redisClient
+}
+
+func initGoRedisClient(options RedisOptions) *redis.Client {
+	return redis.NewClient(&redis.Options{Addr: options.host + ":" + options.port})
+}
+
+func parseFlags() {
+	flag.BoolVar(&useRedis, "useRedis", false, "Specifies whether to use "+
+		"the Redis or Momento cache engine to run operations")
+	host := flag.String("host", "", "Hostname for the Redis server")
+	port := flag.String("port", "", "Hostname for the Redis server")
+	cacheName := flag.String("cacheName", "", "Cache name for the Momento service")
+	defaultTTL := flag.Int("defaultTTLSeconds", 60, "The default TTL for your Momento cache")
+	flag.Parse()
+
+	if useRedis {
+
+		if *host == "" || *port == "" {
+			panic("Running in Redis mode: Redis host (-host) and port (-port) should be provided through command line arguments")
+		}
+		options = &RedisOptions{
+			host: *host,
+			port: *port,
+		}
+	} else {
+
+		if *cacheName == "" {
+			panic("Running in Momento mode: Momento cacheName (-cacheName) should be provided through command line arguments." +
+				"For Redis more, use flag -useRedis along with -host and -port")
```

Comment:
```suggestion
				" For Redis mode, use flag -useRedis along with -host and -port")
```

Reply:
Can you please add a space at the beginning of the "For redis mode..." string? It's being concatenated with the "Running in momento mode..." string but is missing a space between the sentences.